content/kb/troubleshooting-volume-take-long-time-to-mount.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Troubleshooting: Longhorn volumes take a long time to finish mounting"
+author: Phan Le
+draft: false
+date: 2021-02-26
+catelogies:
+  - "csi"
+---
+
+## Applicable versions
+All Longhorn versions.
+
+## Symptoms
+
+When starting workload pod which uses Longhorn volumes,
+observe in Longhorn UI that the Longhorn volumes are attached quickly
+but it takes a long time for the volumes to finish mounting and workload to be able to start.
+
+This issue only happens when Longhorn volumes that have many files/directories and the workload pod sets `fsGroup` in the `securityContext` similar to as below:
```

Comment:
This issue only happens when the Longhorn volumes have many files/directories and `securityContext.fsGroup` is set in the workload pod as shown below:

Reply:
Thank you. Fixed